GitUI/UserControls/RevisionGrid/GraphDrawer/SegmentLanes.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GitUI.UserControls.RevisionGrid.GraphDrawer
+{
+    internal struct SegmentLanes
```

Comment:
Plural type names denote collections, so the name should be singular.
```suggestion
    internal struct SegmentLane
```

With a little refactor, this can be turned into a `readonly struct`. Probably into a ref struct too.